<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>declineStoreItemDlg</class>
 <widget class="QDialog" name="declineStoreItemDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Անկման Քանակը</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbxSubtractCount">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Մնացորդը</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="sbxTotal">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSubtract">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Անկում</string>
         </property>
         <property name="icon">
          <iconset resource="farmmanager.qrc">
           <normaloff>:/FarmManager/Resources/deadbird.png</normaloff>:/FarmManager/Resources/deadbird.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Չեղարկել</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="farmmanager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>declineStoreItemDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSubtract</sender>
   <signal>clicked()</signal>
   <receiver>declineStoreItemDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
